homeassistant/components/climate/device_trigger.py

```diff
@@ -143,6 +143,7 @@ async def async_attach_trigger(
             numeric_state_trigger.CONF_VALUE_TEMPLATE
         ] = "{{ state.attributes.current_temperature }}"
     else:
+        # else means that trigger_type == "current_humidity_changed"
```

Comment:
We often use this pattern:
```suggestion
    else:  # trigger_type == "current_humidity_changed"
```